_api_app/app/Sites/Sections/Entries/SectionEntriesDataService.php

```diff
@@ -745,6 +770,30 @@ public function createEntry($before_entry, $tag)
         ];
     }
 
+    /**
+     * Move entry to other section
+     */
+    public function moveEntry($entry_id, $to_section)
+    {
+        $entries = $this->get();
+        $entry_order = array_search($entry_id, array_column($entries['entry'], 'id'));
```

Comment:
Why is this variable called `entry_order`? Shouldn't it be `entry_id` or `existing_entry_id` or `actual_entry_id`?

Reply:
Renamed to `entryIndex`, this only gets the index position in all entries array for later use.